placebo/pill.py

```diff
@@ -239,17 +245,29 @@ def save_response(self, service, operation, response_data,
         LOG.debug('save_response: %s.%s', service, operation)
         filepath = self.get_new_file_path(service, operation)
         LOG.debug('save_response: path=%s', filepath)
+        if operation in b64_operations.get(service, {}):
+            for key in response_data:
+                if(isinstance(response_data[key], basestring)):
```

Comment:
Are you sure you want to use `basestring`, and not be more specific with `str` or `unicode`?